app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -277,10 +277,11 @@ private void doMenuAction(int menuItemId) {
                 smoothActionBarToggle.runWhenIdle(new Runnable() {
                     @Override
                     public void run() {
-                        startActivity(intent);
                         overridePendingTransition(R.anim.slide_in_left, R.anim.slide_out_left);
                     }
                 });
+                startActivity(intent);
```

Comment:
Yes. The runnable executes the transition. If the intent is called within the runnable then the activity will come after the transition is up. 

You can try it out with the other options on the menu. When you press the button then the page immediately comes up, but as of now if you press Settings, there's a noticeable lag.

No problem though, I can remove it.